jest-config.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+module.exports = (userOptions = {}) => {
+    const {
+        transform = {},
+        moduleNameMapper = {},
+        setupFilesAfterEnv = [],
+        coverageThreshold = {},
+        react = false,
```

Comment:
I think we should name this `enzyme` since React can be tested with many different libraries: Enzyme, React-testing-library, ...